DasharoPayloadPkg/Library/FmpDeviceSmmLib/Flashing.c

```diff
@@ -797,3 +838,320 @@ MergeFirmwareImages (
   FreePool (Data.Updated);
   return NULL;
 }
+
+/**
+  Checks if an image has Boot Guard enabled, by checking is KM and BPM are
+  present in the image.
+
+  @param[in] Image     Image to check
+  @param[in] ImageLen  Size of the image to check
+
+  @return TRUE    If Boot Guard BPM and KM are present
+  @return FALSE   If Boot Guard BPM and KM are absent
+**/
+STATIC
+BOOLEAN
+IsBootGuardEnabled(
```

Comment:
```suggestion
IsBootGuardEnabled (
```

Please keep the project coding style.

There are lots of places where function calls miss the space before `(`

Reply:
fb3363242da4043e98652c308c0ab7727de41c80